Source/WebCore/platform/graphics/avfoundation/objc/LocalSampleBufferDisplayLayer.mm

```diff
@@ -365,7 +365,7 @@ static void setSampleBufferAsDisplayImmediately(CMSampleBufferRef sampleBuffer)
     }
 }
 
-static inline CGAffineTransform videoTransformationMatrix(VideoFrame& videoFrame)
+static inline CGAffineTransform transformationMatrixForVideoFrame(VideoFrame& videoFrame)
```

Comment:
Why does this need to be renamed (in this PR)?

Reply:
like the changelog says, unified sources horror. I added a file and now the two `videoTransformationMatrix` methods end up in the same unified source and conflict :)